gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -537,6 +538,9 @@ subroutine Initialize ( gc, import, dumexport, clock, rc )
     call ESMF_ConfigGetAttribute(config, value=cFileOrder,         &
                                          label='FileOrder:', default='ABC', rc=status)
     _VERIFY(STATUS)
+    call ESMF_ConfigGetAttribute(config, value=intState%check_if_file_exists,  &
+                                         label='Check_If_File_Exists:', default=.true., _RC)
```

Comment:
```suggestion
                                         label='Fail_If_File_Exists:', default=.true., _RC)
```
This is the name that really matters.   Probably should have a bit of discussion.   In general, the default for new options should be `.false.` if we can arrange it.
Variants:

-    `NO_CLOBBER`  (default true)
-    `CLOBBER` (default false).

Reply:
@tclune Made all the changes, what does no_clobber/clobber have to do with this PR? This has nothing to do with clobber here in history. And the PFIO Server is not clobbering and has never been clobbering now if the file is there, it's just writing whether the file is good or not, if it's bad well it will crash without more fixes in pFIO which I think I see how to do at least track and take action, modifying the API is more working since there are a lot.